src/ax/saga/mapping/keyset/mapper.go

```diff
@@ -0,0 +1,65 @@
+package keyset
+
+import (
+	"context"
+
+	"github.com/jmalloc/ax/src/ax"
+	"github.com/jmalloc/ax/src/ax/persistence"
+	"github.com/jmalloc/ax/src/ax/saga"
+)
+
+// Mapper is an implementation of saga.Mapper that maps messages to saga
```

Comment:
Something doesn't quite flow here?
> that maps messages to saga using disjoint

I could be wrong, but should it be *saga's* or *saga instance* ?